extensions/amp-story/1.0/test/test-amp-story-request-service.js

```diff
@@ -18,11 +18,20 @@ describes.fakeWin('amp-story-request-service', {amp: true}, (env) => {
     xhrMock = env.sandbox.mock(requestService.xhr_);
   });
 
-  it('should not load the share config if no attribute is set', async () => {
+  it('should fall back to inline config if no src is set', async () => {
```

Comment:
Testing the fallback sounds good but we should not remove the test for when there is no configuration at all